public/sandbox/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>OWID GCharts</title>
+        <link
+            href="https://fonts.googleapis.com/css?family=Lato:300,400,400i,700,700i|Playfair+Display:400,700"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" href="http://localhost:8090/css/commons.css" />
+        <link rel="stylesheet" href="http://localhost:8090/css/owid.css" />
+    </head>
+    <body class="ChartPage">
+        <main>
+            <figure data-grapher-src=""></figure>
+        </main>
+        <script src="http://localhost:8090/js/commons.js"></script>
+        <script src="http://localhost:8090/js/owid.js"></script>
+        <script src="chart.js"></script>
```

Comment:
```suggestion
        <script src="chart.sample.js"></script>
```

Reply:
Ah I should document this. This one is sort of like ".env.example" in that I expect you to create a chart.js yourself (from a freeze).